src/components/Portfolio/LoanFilterBar.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+	<div>
+		<div class="tw-flex tw-flex-col">
+			<div class="tw-flex tw-justify-between tw-mb-2">
+				<div class="tw-text-sm tw-text-h2">
+					Filter loans
+				</div>
+				<kv-button
+					class="tw-text-sm"
+					variant="primary"
+				>
+					Export {{ totalLoans }} loans
+				</kv-button>
+			</div>
+			<kv-text-input
+				id="loan-filter-text-input"
+				v-model="searchText"
+				placeholder="Search by name, ID, partner or location"
+			/>
+			<div class="tw-flex tw-flex-wrap tw-items-center tw-gap-2 tw-mt-2">
+				<div class="tw-flex tw-items-center tw-gap-2">
+					<span class="tw-text-secondary">Status:</span>
+					<kv-select
+						id="loan-filter-select"
+						v-model="selectedStatus"
+					>
+						<option value="all">
+							All loans
+						</option>
+					</kv-select>
+				</div>
+				<div class="tw-flex tw-items-center tw-gap-2">
+					<span class="tw-text-secondary">Filter by:</span>
+					<kv-select
+						v-model="selectedLocation"
+					>
+						<option value="all">
+							Location
+						</option>
+					</kv-select>
+					<kv-select
+						v-model="selectedPartner"
+					>
+						<option value="all">
+							Partner
+						</option>
+					</kv-select>
+				</div>
+			</div>
+		</div>
+		<div class="tw-text-secondary tw-mt-2">
+			Sorted by date posted on Kiva
+		</div>
+	</div>
+</template>
+
+<script>
+import { KvSelect, KvTextInput, KvButton } from '@kiva/kv-components';
+
+export default {
+	name: 'LoanFilterBar',
+	components: {
+		KvSelect,
+		KvTextInput,
+		KvButton,
+	},
+	data() {
+		return {
```

Comment:
It looks like filters in general aren't implemented yet?

Reply:
Right.